### PR TITLE
Add georgian locales

### PR DESCRIPTION
**Pull request by lamartire**
At ISO 639-1:2002 georgian language has `ka` code.

### CODE REVIEW

File: src/locale/ka.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import dayjs from 'dayjs'
+
+const locale = {
+  name: 'ka',
+  weekdays: 'კვირა_ორშაბათი_სამშაბათი_გარემო_ხუთშაბათი_პარასკევი_ამ შაბათს'.split('_'),
```
[the remark this answers is not in the document]
Comment (lamartire) [reply src/locale/ka.js]: Yeah, you right.

**Comment by lamartire** [comment]
@iamkun hey, when do you will release lib with this pull request? :smile_cat:

**Comment by lamartire** [comment]
Very need it.

**Comment by iamkun** [comment]
@lamartire Maybe in this week I'm afraid. You may could copy the locale file to your project to use it at the moment.

**Comment by lamartire** [comment]
Sure, I see. I look forward to it :sparkles:

**Pull request merged**